Sources/SFSafeSymbols/ImageExtension.swift

```diff
@@ -0,0 +1,14 @@
+import SwiftUI
+
+@available(iOS 13.0, *)
+public extension Image {
+
+    /// Creates a instance of `Image` with a `UIImage` object that has its rendering mode set to `.alwaysTemplate`.
+    ///
+    /// - systemSymbol: The `SFSymbol` describing this image.
+    @available(iOS 13.0, *)
+    init(systemSymbol: SFSymbol) {
+        self.init(uiImage: systemSymbol.toImage.withRenderingMode(.alwaysTemplate))
```

Comment:
Just wondering, whether we should use the `.alwaysTemplate` option everywhere... or make it accessible as an optional param? However, probably 100 % of the time, we want a template image, do you agree? If yes, would you mind implementing this everywhere? ;)

Reply:
Just had a look at the WWDC session about SF Symbols and they said all symbols are already renderer as `.alwaysTemplate`, so we won't need this ;)

Reply:
When I tried in in Xcode with preview it didn't work, until I set the rendering mode to `.alwaysTemplate`. I'm watching the session right now. Let me get back to you afterwards.